### DISCUSSION TITLE
Request for help with error occurring while testing Ray tracing tutorial

### DISCUSSION THREAD

**Discussion by hyowon2075** [NVlabs/sionna#552] [Q&A]
Dear everyone,

I recently found out about SIONNA and am following the tutorial.

SIONNA is installed on WSL2 Ubuntu, and tensorflow version 2.15 is installed together.

While following [Ray tracing_introduction](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html), an error occurred in the first line of the [11] code:

`paths = scene.compute_paths(max_depth=5,
                            num_samples=1e6) `

The detailed error is as follows:
![image](https://github.com/user-attachments/assets/9814f84e-0942-48a7-9d1f-2538e58ce34f)
![image](https://github.com/user-attachments/assets/34b5bd50-294f-45d4-96cf-bf843e6c931c)

The exception occurs in other ray tracing examples as well. 

Could this exception be related to LLVM-backend? 

If any users have overcome this error, I would appreciate your help.

Added: I downgraded and reinstalled the gpu driver to v531.79, referring to the [previous issue regarding compute_path()](https://github.com/NVlabs/sionna/issues/202), but I'm still having the same issue.
Nvidia drive version: 531.79
CUDA: 12.1
LLVM: 14
Mitsuba: 3.5.2
Drjit: 0.4.6

**Comment by merlinND** [comment]
Hello @hyowon2075,

Please see this existing discussion about WSL: https://github.com/NVlabs/sionna/discussions/522#discussioncomment-10111917
The main issue is that AFAIK, OptiX is not supported in WSL. OptiX is the library that Mitsuba uses to do ray tracing.

The solution is to install Sionna in Windows directly, not WSL.

**Reply by hyowon2075** [reply]
Dear @merlinND,

Thank you for your kind reply.

I will check it after installing in Windows environment.
